utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -28,7 +28,8 @@ case class KafkaConfig(
     schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig(),
     avroAsJsonSerialization: Option[Boolean] = None,
     kafkaAddress: Option[String] = None,
-    idleTimeout: Option[IdlenessConfig] = None
+    idleTimeout: Option[IdlenessConfig] = None,
+    deliveryGuarantee: Option[DeliveryGuarantee.Value] = None
```

Comment:
Maybe it would be better to stick with Flink naming convention (`producerSemantic` or `sinkSemantic`)? Thanks to that, a person who knows Flink will be sure that this property is just exposing what is available in Flink.